docs/user_manual/components.md

```diff
@@ -695,15 +695,116 @@ A `fault` has no steady state output.
+| `status`           | `int8_t`  | -    | connection status to the regulated object | &#10004; | &#10004; |         `0` or `1`          |
+
+### Transformer tap regulator
+
+* type name: `transformer_tap_regulator`
+* base: {hoverxreftooltip}`user_manual/components:regulator`
 
+`transformer_tap_regulator` defines a regulator for transformers in the grid.
+At the time of writing, a transformer tap regulator regulates a component that is either a {hoverxreftooltip}`user_manual/components:transformer` or a {hoverxreftooltip}`user_manual/components:Three-Winding Transformer`.
+
+The transformer tap regulator overloads the `tap_pos` of the transformer it regulates in the range set by the user via `tap_min` and `tap_max` (i.e., `(tap_min <= tap_pos <= tap_max)` or `(tap_min >= tap_pos >= tap_max)`).
+It regulates the tap position so that the voltage on the control side is in the chosen voltage band.
+Other points further into the grid on the control side, away from the transformer, can also be regulated by providing the cumulative impedance across branches to that point as an additional line drop compensation.
+This line drop compensation only affects the controlled voltage and does not have any impact on the actual grid. It may therefore be treated as a virtual impedance in the grid.
+
+#### Input
+
+| name                       | data type                                                                                                                                                                                                                                                                                                          | unit     | description                                                                                             |                         required                         |  update  |   valid values    |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | ------------------------------------------------------------------------------------------------------- | :------------------------------------------------------: | :------: | :---------------: |
+| `control_side`             | {py:class}`BranchSide <power_grid_model.enum.BranchSide>` if the regulated object is a {hoverxreftooltip}`user_manual/components:transformer` and {py:class}`Branch3Side <power_grid_model.enum.Branch3Side>` if it the regulated object is a {hoverxreftooltip}`user_manual/components:Three-Winding Transformer` | -        | the controlled side of the transformer                                                                  | &#10024; only for power flow with automatic tap changing | &#10004; |                   |
```

Comment:
Why are `u_set` and `u_band` required, but is `control_side` only needed when you want to perform a power flow with automatic tap?

Reply:
you're right